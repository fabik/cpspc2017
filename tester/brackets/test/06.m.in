200 215 14 73
50 61 }
145 132 ]
14 126 }
84 192 {
42 141 }
136 176 ]
199 77 )
48 128 >
85 178 >
14 188 )
151 80 ]
14 194 {
141 115 )
182 146 >
169 129 }
100 196 }
122 14 [
59 34 >
21 118 }
77 13 (
32 84 )
14 63 )
147 8 }
144 104 )
81 100 ]
183 185 <
126 67 )
139 18 }
96 7 [
37 163 )
123 70 >
55 14 [
102 87 ]
14 138 ]
121 46 )
57 74 >
45 91 }
72 62 )
98 158 )
29 14 <
93 167 }
152 51 )
5 45 >
23 66 )
142 14 (
92 59 ]
52 49 >
16 103 )
153 14 (
71 114 }
41 40 >
14 174 >
69 173 )
148 155 )
159 14 <
110 97 }
163 68 ]
90 78 }
80 81 }
12 23 ]
49 5 }
187 14 [
14 164 >
83 152 >
79 122 (
162 116 ]
164 191 >
18 50 }
173 11 )
54 99 >
157 83 )
99 39 ]
188 154 ]
129 3 )
186 177 )
171 119 }
39 131 >
27 14 {
8 96 ]
103 184 ]
165 33 >
25 12 )
105 109 >
24 170 >
127 133 ]
138 181 )
20 165 >
149 4 )
14 95 }
115 6 }
114 43 )
6 71 )
76 180 ]
40 24 ]
179 113 ]
180 159 [
94 35 }
116 190 ]
125 117 )
118 110 ]
143 14 (
194 14 {
75 19 )
67 98 }
177 157 >
88 145 >
47 187 {
63 56 >
119 30 ]
91 161 >
112 166 ]
10 112 ]
196 130 ]
61 142 [
26 125 >
60 58 }
14 195 ]
56 160 )
44 189 )
133 10 ]
181 25 ]
132 172 >
124 121 }
14 186 >
137 92 >
170 137 ]
175 20 >
74 156 }
109 55 <
193 197 )
190 101 ]
178 175 >
154 168 )
51 148 >
66 149 ]
43 31 }
104 134 }
14 88 ]
158 144 }
70 48 )
184 76 )
146 9 }
19 79 )
130 120 }
17 54 ]
34 38 (
28 86 <
172 41 ]
97 147 ]
160 107 >
135 171 ]
113 17 >
14 140 >
176 151 }
22 69 )
197 32 >
58 89 }
191 198 >
168 72 ]
1 179 >
46 199 }
174 52 }
106 42 )
108 73 >
65 94 >
33 108 >
89 139 }
36 105 }
62 16 ]
3 106 }
131 47 ]
7 14 [
53 183 }
120 143 (
15 123 )
11 200 )
38 14 {
189 64 )
107 26 )
2 44 )
9 36 >
185 14 (
111 21 ]
64 75 )
117 15 >
82 85 >
87 29 ]
195 136 }
200 2 )
30 111 }
14 169 )
14 135 }
128 153 {
31 27 [
78 60 }
150 182 }
95 90 }
140 1 ]
161 57 }
4 37 ]
14 22 )
13 14 <
86 14 {
68 28 )
14 127 ]
192 14 <
156 53 >
35 93 >
198 82 >
134 124 )
101 102 ]
14 65 }
155 193 >
167 150 >
166 162 ]
