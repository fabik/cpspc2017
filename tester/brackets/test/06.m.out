12439474748376
